desc/geometry/curve.py

```diff
@@ -921,8 +945,57 @@ def method(self, new):
                 + f"instead got unknown method {new} "
             )
 
+    def compute(
```

Comment:
why does this need to be redefined from the base class?

Reply:
This was the whole problem with getting the `method` parameter to work as a kwarg because it didn't work when we added it to `transforms`. I followed what Daniel did in his rpz basis pr